README.md

```diff
@@ -6,9 +6,13 @@ BaSta-LedControl is an Arduino program used to control the red and yellow basket
 
 BaSta-LedControl is intended to be used on an Arduino Uno board. The Stramatel basketball protocol is received through serial input. Two digital output pins control the LED state e.g. via switch relays.
 
+The Rx+ and the Rx- Pin of the Stramatel Remote is connected with a MAX485 TTL to RS485 Switch.
+
 To keep the USB serial interface enabled, the protocol is received through a separate Rx-pin and the SoftwareSerial plugin. Following is the pin usage for an Arduino Uno board:
 
-- 2 - Rx serial input (SoftwareSerial)
-- 3 - gpio output #1 (red stripe)
-- 4 - gpio output #2 (yellow stripe)
-- 5 - Tx serial input (SoftwareSerial - not used)
\ No newline at end of file
+- 2 - gpio output #1 (yellow stripe)
+- 3 - gpio output #2 (red stripe)
+- 4 - Rx serial input (SoftwareSerial)
+- 5 - ReceiverEnablePin (LOW);
+- 6 - DriverEnablePin (LOW);
+- 7 - Tx serial input (SoftwareSerial - not used)
```

Comment:
These changes are correct, thank you! I didn't update the readme after/with my last commit that changed the state to the latest hardware usage that worked for me.